streams-tests/jvm/src/test/scala/zio/stream/experimental/ZStreamPlatformSpecificSpec.scala

```diff
@@ -103,7 +103,6 @@ object ZStreamPlatformSpecificSpec extends ZIOBaseSpec {
             stream = ZStream.asyncZIO[Any, Throwable, Int](
                        cb => {
                          global.execute { () =>
-                           println("Rawr")
```

Comment:
😮

Reply:
If it was there for a reason I can revert this change

Reply:
I'm sure there wasn't :)